docs/getting-started/new_project_guide.md

```diff
@@ -87,6 +88,14 @@ You project's homepage.
 The primary contact and list of other contacts to be CCed. Each person listed gets access to ClusterFuzz, including crash reports and fuzzer statistics, and are auto-cced on new bugs filed in the OSS-Fuzz
 tracker. If you're a primary or a CC, you'll need to use a [Google account](https://support.google.com/accounts/answer/176347?hl=en) to get full access. ([why?]({{ site.baseurl }}/faq/#why-do-you-require-a-google-account-for-authentication)).
 
+### vendor_ccs (optional) {#vendor}
+The list of vendor email addresses that are downstream consumers of the project and want access to
+the bug reports as they are filed.
+
+Any changes to this list must follow these rules:
+- Should be approved by the project maintainer (GitHub PR, project mailing list, etc).
+- Should be an organization email address.
```

Comment:
suggestion: "An organization email address is used."

Reply:
Done!